src/templates/OutdatedBrowser.html

```diff
@@ -0,0 +1,9 @@
+<div class="outdated-browser">
+  <p>
+    Sorry, you are using an outdated browser.
+  </p>
+  <p>
+    Please <a href="http://outdatedbrowser.com/en">upgrade to a newer version</a>
```

Comment:
Also, probably not much of an issue, but realized that this site isn't the greatest for mobile browsers... not sure there is actually much to be done in that case though.